devcontainer.Dockerfile

```diff
@@ -0,0 +1,11 @@
+FROM golang:1.18.5
```

Comment:
Is `1.18.5` the correct Golang version?

Reply:
This is ok. Most people are using a flavor of 1.18 anyways. 

We're also investing in a monorepo, which will make this kind of standardization a lot easier.